ReactiveCocoa/Swift/Signal.swift

```diff
@@ -694,12 +704,12 @@ extension SignalType {
 			let otherState = CombineLatestState<U>()
 			
 			let onBothNext = { () -> () in
-				sendNext(observer, (signalState.latestValue!, otherState.latestValue!))
+				observer.sendNext((signalState.latestValue!, otherState.latestValue!))
 			}
 			
-			let onError = { sendError(observer, $0) }
-			let onBothCompleted = { sendCompleted(observer) }
-			let onInterrupted = { sendInterrupted(observer) }
+			let onError = { observer.sendError($0) }
```

Comment:
I wonder if you can just write `let onError = observer.sendError` here and everywhere else.